ast/util.go

```diff
@@ -1,6 +1,35 @@
+// Copyright 2017 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
 package ast
 
-import "strings"
+import (
+	"strings"
+)
+
+type NodeTextCleaner struct {
```

Comment:
Is `NodeTextCleaner` used in any other packages ?

Reply:
no,only string